torchmultimodal/architectures/omnivore.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+from torch import nn
+
+
+class OmnivoreArchitecture(nn.Module):
```

Comment:
I know you are following the other examples here. but omnivorearchitecture is essentially an encoder+ head. Idk if we need a separate "architecture" for it. we can discuss in design meeting

Reply:
Yeah, the omnivore architecture is basically an encoder + multiple heads (using ModuleDict). Sure, it is a good idea to discuss on design meeting

Reply:
Have talk offline, the OmnivoreArchitecture is indeed quite generic like MTL architecture. However for now we will keep it as Omnivore and if there are other use cases who need MTL we can reuse and make it more generic.